pkgs/build-support/fetchurl/boot.nix

```diff
@@ -4,12 +4,13 @@ let mirrors = import ./mirrors.nix; in
 
 { url ? builtins.head urls
 , urls ? []
-, sha256
+, sha256 ? ""
+, hash ? ""
 , name ? baseNameOf (toString url)
 }:
 
 import <nix/fetchurl.nix> {
-  inherit system sha256 name;
+  inherit system hash sha256 name;
```

Comment:
https://github.com/NixOS/nix/blob/master/src/libexpr/fetchurl.nix is not checking if both are set. We should assert that here.